pkg/apis/kops/cluster.go

```diff
@@ -512,6 +512,9 @@ type EtcdManagerSpec struct {
 	// This allows etcd setting to be overwriten. No config validation is done.
 	// A list of etcd config ENV vars can be found at https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/configuration.md
 	Env []EnvVar `json:"env,omitempty"`
+	// LogVerbosity allows the klog library verbose log level to be set. The default is 6.
+	// https://github.com/google/glog#verbose-logging
+	LogVerbosity *int32 `json:"logVerbosity,omitempty"`
```

Comment:
Can we rename this to LogLevel to follow the pattern used in other components.

https://github.com/kubernetes/kops/search?q=logLevel

Reply:
Sure thing - also, to avoid further confusion it might help to clarify further that this is the etcd-manager log level, not the etcd it manages; any suggestions on that? e.g. a clearer comment (which I saw is put into the CRD), but maybe also in the property name?

Reply:
I think just a comment mentioning this is for etcd-manager would be enough.